armi/cases/case.py

```diff
@@ -716,31 +716,44 @@ def writeInputs(self, sourceDir: Optional[str] = None):
 
 
 def copyInputsHelper(
-    fileDescription: str, fileFullPath: pathlib.Path, destPath: pathlib.Path
+    fileDescription: str,
+    sourcePath: str,
+    destPath: str,
+    origFile: str,
 ) -> str:
     """
     Helper function for copyInterfaceInputs: Creates an absolute file path, and
-    copies the file to that location.
+    copies the file to that location. If that file path does not exist, returns
+    the file path from the original settings file.
 
     Parameters
     ----------
     fileDescription : str
         A file description for the copyOrWarn method
 
-    fileFullPath : pathlib.Path object
+    sourcePath : str
         The absolute file path of the file to copy
 
-    destPath : pathlib.Path object
+    destPath : str
         The target directory to copy input files to
 
+    origFile : str
+        File path as defined in the original settings file
+
     Returns
     -------
-    destFilePath : str
+    destFilePath (or origFile) : str
     """
-    sourceName = os.path.basename(fileFullPath.name)
-    destFilePath = os.path.abspath(destPath / sourceName)
-    pathTools.copyOrWarn(fileDescription, fileFullPath, destFilePath)
-    return destFilePath
+    sourceName = os.path.basename(sourcePath)
+    destFilePath = os.path.join(destPath, sourceName)
+    try:
+        pathTools.copyOrWarn(fileDescription, sourcePath, destFilePath)
+        if pathlib.Path(destFilePath).exists():
+            return destFilePath
+        else:
+            return origFile
+    except Exception:
```

Comment:
Not sure what this should be here. `copyOrWarn` throws its own exception, but I think it's still important to explicitly check `destFilePath` exists. But that can sometimes fail depending on the operating system (see `OSError` in new-line 841.)

Reply:
Sometimes on certain really old computing clusters, that will remain nameless, copying a file might take a really long time.  And so checking if the file exists gets hard, because Python might think the copy took a long time, or sometimes Python might think the copy is done a few seconds before it really is.

I think it's clean enough as it is.  At least, I can't think of any clever way to make it cleaner.